pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractJUnitRule.java

```diff
@@ -22,42 +22,14 @@
 
 public abstract class AbstractJUnitRule extends AbstractJavaRule {
 
-    public static final Class<?> JUNIT3_CLASS;
-    private static final String JUNIT3_CLASS_NAME = "junit.framework.TestCase";
-    public static final Class<?> JUNIT4_CLASS;
-    private static final String JUNIT4_CLASS_NAME = "org.junit.Test";
-    public static final Class<?> JUNIT5_CLASS;
-    private static final String JUNIT5_CLASS_NAME = "org.junit.jupiter.api.Test";
+    protected static final String JUNIT3_CLASS_NAME = "junit.framework.TestCase";
+    protected static final String JUNIT4_CLASS_NAME = "org.junit.Test";
+    protected static final String JUNIT5_CLASS_NAME = "org.junit.jupiter.api.Test";
 
     private boolean isJUnit3Class;
     private boolean isJUnit4Class;
     private boolean isJUnit5Class;
 
-    static {
-        Class<?> c;
-
-        try {
-            c = Class.forName(JUNIT3_CLASS_NAME);
```

Comment:
I guess, we have the same problem in [MoreThanOneLoggerRule](https://github.com/pmd/pmd/blob/70fc3d75bb018a333c4612fc5f37446d43b32844/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/errorprone/MoreThanOneLoggerRule.java#L32)...

Reply:
I'll adjust this class the same way.

Reply:
Definitely! Good catch!